app/server/methods/residents.js

```diff
@@ -70,29 +70,32 @@ Meteor.methods({
     //  sort in reverse order by activity date
     return Activities.find({'residentIds': residentId, activityDate: {$gte: twoWeeksAgo, $lte: now}}, {sort : {activityDate:  -1} });
   },
-  getResidentActivitiesByTypeLast30Days (residentId, activityTypeId) {
-    // Date thirty days ago
-    var thirtyDaysAgo = moment().subtract(30, "days").toDate();
+  'getResidentRecentActivitiesByType': function (residentId, activityTypeId, date) {
+    // Date two weeks ago
+    let startDate = moment().subtract(4, "weeks").toDate();
```

Comment:
Use `const` if the reference will not change at a later point.

Reply:
doesnt the reference change at the line 77?
````
    if(date) {
      startDate = date;
      endDate = moment(date).endOf('month').toDate();
    }